sw/airborne/firmwares/rotorcraft/stabilization/stabilization_indi.c

```diff
@@ -193,6 +210,12 @@ void stabilization_indi_init(void)
   //Calculate G1G2_PSEUDO_INVERSE
   calc_g1g2_pseudo_inv();
 
+  // Initialize the array of pointers to the rows of g1g2
+  uint8_t i;
+  for(i=0; i<INDI_OUTPUTS; i++) {
```

Comment:
The dimension of g1g2 is outputs * num_act. Bwls is an array of pointers to the rows of g1g2, so size INDI_OUTPUTS. So this for loop just has to loop over the number of outputs, and the number of actuators can be something else.